code/game/objects/items.dm

```diff
@@ -196,6 +196,8 @@
 		if(src == user.get_inactive_hand())
```

Comment:
Does this work with the multi hand systems? It seems like you'd need to check if its being held, and also that its not the active hand instead.